### PR TITLE
Implement porting efforts from ZD5241 and ZD5815

### PR DESCRIPTION
**Pull request by kaleb-himes**
Bringing EBSNET support up-to-date for the latest effort. See notes in ZD5241 and ZD5815

### CODE REVIEW

File: wolfcrypt/src/wc_port.c [hunk 1]
```diff
@@ -1175,6 +1175,19 @@ int wolfSSL_CryptHwMutexUnLock(void) {
         return 0;
     }
 
+    int ebsnet_fseek(int a, long b, int c)
```
Comment (dgarske) [inline wolfcrypt/src/wc_port.c]: This function will need to be defined in a header... ideally wc_port.h above the `#define XFSEEK                   ebsnet_fseek`

Comment (kaleb-himes) [reply wolfcrypt/src/wc_port.c]: I'm highly confident this is prototyped in vfile.h (the EBSnet file system API header) but I am happy to confirm that with the customer.

Comment (kaleb-himes) [reply wolfcrypt/src/wc_port.c]: @dgarske I'll CC you on the email.

File: wolfssl/wolfcrypt/settings.h [hunk 2]
```diff
@@ -794,7 +794,24 @@ extern void uITRON4_free(void *p) ;
...
+    #define WOLFSSL_HAVE_MAX
+    #define WOLFSSL_HAVE_MIN
+
+    #define USE_FAST_MATH
+    #define TFM_TIMING_RESISTANT
+    #define WC_RSA_BLINDING
+    #define ECC_TIMING_RESISTANT
+
+    #define HAVE_ECC
+    #define WOLFSSL_STATIC_RSA
```
Comment (cconlon) [inline wolfssl/wolfcrypt/settings.h]: Was this intended, to enable static RSA for all EBSNET builds?

**Pull request merged**